src/System.Diagnostics.FileVersionInfo/src/System.Diagnostics.FileVersionInfo.builds

```diff
@@ -9,6 +9,11 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Diagnostics.FileVersionInfo.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netcore50</TargetGroup>
```

Comment:
True.  We should actually see about extending Ref.Metadata to read the version resource from the PE.  Or maybe better would be to ask Windows to add GetFileVersionInfoEx to UWP.